dev/build/windows/configure_profile.sh

```diff
@@ -17,27 +17,30 @@ donefile=~/.bash_profile.upated
 if [ ! -f $donefile ] ; then
 
     echo >> $rcfile
-    
-    if [ "$1" != "" -a "$1" != " " ]; then
-      echo export http_proxy="http://$1" >> $rcfile
-      echo export https_proxy="http://$1" >> $rcfile
-      echo export ftp_proxy="http://$1" >> $rcfile
+
+    if [ "$1" != "" ] && [ "$1" != " " ]; then
+      {
+        echo export http_proxy="http://$1"
+        echo export https_proxy="http://$1"
+        echo export ftp_proxy="http://$1"
+      } >> $rcfile
```

Comment:
AFAICT all echoing is redirected to rcfile so you could as well do `exec >> $rcfile` at the beginning.

Reply:
@SkySkimmer: done, there's already a `echo >> $rcfile` of which I don't understand the purpose.